### ISSUE TITLE
The BFD.  So you think you like Delays... RUN!

### ISSUE THREAD

**Issue by mschack** [mschack/VCV-Rack-Plugins#37]
This is a mock-up.  I admit I may have gone too far... I am not promising anything but tell me what you think.  I may need to strip it down a bit.  This shows 6 delays, 6 pans, 10 LFOs and 3 filters of which you can hook up in any merry configuration you please.  We here at mscHack like to give the end user the rope to hang themselves with...  Here is lots of rope.

![image](https://user-images.githubusercontent.com/32986033/34073384-a28375a0-e266-11e7-94ec-9b6541527612.png)

**Comment by blackearth2014** [comment]
Don’t touch a thing. This is the jump off of all jump offs. This module will be the most used delay module period. Let me hang myself with it.

**Comment by blackearth2014** [comment]
I had posted a mark upon Facebook and asked your questions about the features. The latest consensus agreed with me. Keep everything.

**Comment by blackearth2014** [comment]
just ask him to label things to make it more clear how it works, but of course the more options, the better (y) very cool module !

From Facebook

**Comment by mschack** [comment]
Thanks, I'll see what I can do

**Comment by mschack** [comment]
Might take me longer to figure out how it works than to develop it in the first place...

**Comment by blackearth2014** [comment]
oh no, if you are inspired by Ohmforce you MUST add stickers to the plate!!!!

From Facebook

**Comment by jamesbernard** [comment]
I am all for a full featured multi-delay unit.. but I would like to see you develop this one a bit more. The ins/out need to be labeled clearly (as do the delay time knobs). It would be great to be able to tap audio in before and after the feedback loop of each delay unit and also have a built in side chain/ducking for the delayed signal. Also.. not seeing a wet/dry mix or feedback amount. Some soft saturation on the feedback would be nice as well.

**Comment by blackearth2014** [comment]
I think the Lvl input represents the feedback and the LFO out the Lvl in will give the results you are asking plus having the ability to be sync.

**Comment by jamesbernard** [comment]
That reply makes no sense at all. The jack below the level knob is CV and would not be the same as tapping into the feedback loop audio path. Are you the developer?

**Comment by mschack** [comment]
Yeah there is definitely going to be some experimentation on this one.  Suggestions are appreciated.  Forgive the crude drawing, I mocked it up in MS Paint...

**Comment by mschack** [comment]
What I should do is try and mock this up in VCV using existing delays to see what works and what doesn't

**Comment by blackearth2014** [comment]
Right, the Lvl port is CV. What was I looking at the time I wrote that, I have no clue.

**Comment by blackearth2014** [comment]
Do the pan need to have all those ports? Why not just have one pan knob & CV port for each Delay. This will regain more space for needed features such as dry/wet/CV mix, feedback/CV and others.

**Comment by mschack** [comment]
My thought was that each delay was mono and you could route any to one or more of the pan ports.  There is no actual left or right in this until you feed the ouputs.  The pan ports may be excessive and would probably be the first thing I cut down on.

**Comment by jamesbernard** [comment]
Also.. one thing I noticed about your Ping Pong Delay..  The sync seems to be a bit off when switching from normal to reverse and it would be nice (again) to have some idea of the Delay times that are being dialed in with the knobs.. right now it's just "use your ears" method. Granted, I can work with that but others might have more difficulty dialing in the right setting. Also the mixers in this module pack get into digital distortion fairly quickly. It would be nice to have some option of soft clip on the channels/master to prevent that, as it's not a "pleasing" distortion.

**Comment by blackearth2014** [comment]
I agree with this "The pan ports may be excessive and would probably be the first thing I cut down on." & "mixers in this module pack get into digital distortion fairly quickly. It would be nice to have some option of soft clip on the channels/master to prevent that, as it's not a "pleasing" distortion."

**Comment by mschack** [comment]
My development now has avoided any text feedback, frankly because I have never looked into it.  This will change though.  I also tend to cram too much stuff together and have no room left to put anything :)  I have actually been pondering changing the GUI completely and using some sort of matrix as opposed to having to hook up all those wires, thoughts?

**Comment by mschack** [comment]
I do appreciate the feedback, as I have said I am no expert here...

**Comment by jamesbernard** [comment]
YES to a matrix. Wires are nice and it would be good to have the option to jack into that matrix. I love your stuff and use it in every patch I make.. so my feedback is coming from someone who wants these modules to be amazing. I have a lot of experience working with many different GUI for both hardware and software (worked for Korg, Propellerhead and now Spectrasonics).. so I would love to help out where I can.

**Comment by mschack** [comment]
Oh that's really cool.  My software GUI experience is a little more mundane, I worked in the industrial field.  Mostly did firmware though.

**Comment by Amidamaru888** [comment]
Oh!!! this is how delayed love feel like, I don't really understand most of modules, but I love to spend hours and hours developing new sounds. My racks starts (allmost all the time) with your sequencers Mshack!!! Continue with your amazing work please!!!

**Comment by mschack** [comment]
Thanks for the comment @Amidamaru888

**Comment by blackearth2014** [comment]
MSCHack,

What do you think of a layout like this to inspire your update monster delay module?

<img width="503" alt="noisesx3" src="https://user-images.githubusercontent.com/7956901/34143463-d523b286-e45a-11e7-8c81-46e7c1744706.png">

**Comment by mschack** [comment]
That looks cool.

**Comment by mschack** [comment]
I've been pondering something like a stepped delay.  Like the APP but repeating sounds instead of notes.

**Comment by LKHSogpit** [comment]
That sync LFO interests me!
